### PR TITLE
Fix/export colors dictionary correctly

### PR DESCRIPTION
**Pull request by codingwithdani**
## Category/Component

**TASK**: 

### Types of changes

- [x] 🪲 Bug fix (non-breaking change which fixes an issue)
- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 🧾 Documentation
- [x] 📷 Demo
- [ ] 🧪 Test
- [ ] 🧠 Refactor
- [ ] 💄 Styles

### Description, Motivation and Context

On the Button component, the array "atomButtonColors" it is not working correctly and in no vertical is it being used. 

It's only implemented in some cases and it's working because it's taking the default color "primary":
https://github.mpi-internal.com/search?q=atomButtonColors&type=code

To fix this issue I changed the array "atomButtonColors" for an Object.

We are going to be able to use the atomButtonColors variable on all verticals and remove magic strings.

### CODE REVIEW

**Review by quinwacca — APPROVED** [review]

**Comment by andresin87** [comment]
🚨 This is a breaking change. Can we discuss why we need to make this change or maybe e can find a soft approach

File: components/atom/button/test/index.test.js [hunk 1]
```diff
@@ -542,15 +542,16 @@ describe(json.name, () => {
   })
 
   describe('atomButtonColors', () => {
```
Comment (andresin87) [inline components/atom/button/test/index.test.js:544]: check the other exported object tests and copy them

Comment (codingwithdani) [reply components/atom/button/test/index.test.js:544]: solved

File: components/atom/button/test/index.test.js [hunk 2]
```diff
@@ -542,41 +542,45 @@ describe(json.name, () => {
...
-    it('value must be an array', () => {
+    it('value must be a dictionary', () => {
```
Comment (andresin87) [inline components/atom/button/test/index.test.js]:
```suggestion
    it('value must be an object enum', () => {
```

File: components/atom/button/test/index.test.js [hunk 3]
```diff
@@ -542,41 +542,45 @@ describe(json.name, () => {
   })
 
   describe('atomButtonColors', () => {
-    it('value must be an array', () => {
+    it('value must be an object enum, () => {
```
Comment (andresin87) [inline components/atom/button/test/index.test.js]:
```suggestion
    it('value must be an object enum', () => {
```

**Review by andresin87 — APPROVED** [review]

**Pull request merged**